### ISSUE TITLE
Disable Anybar by default

### ISSUE THREAD

**Issue by ArondeParon** [mikeerickson/phpunit-pretty-result-printer#122]
Our CI systems started generating exceptions when using the `0.20.*` build. We downgraded to `0.19.14` and noticed that the error was resolved.

When comparing the differences between these versions, the biggest change is the introduction of Anybar support. 

I don't have Anybar installed, and tests seem to be working fine when running them locally on my Mac Os machine, but when running them in our CI environment (Ubuntu), we get unexpected results.

**TL;DR:** it would probably be wise to disable it by default in the configuration.

See also https://github.com/laravel/framework/issues/6322#issuecomment-425367389

**Comment by mikeerickson** [comment]
My sincerest apologies, I didn’t know there would be any harm.  I will post a new patch today to mark this as false by default

**Comment by mikeerickson** [comment]
@ArondeParon Hey there, just wanted to let you know, I have tagged a new release `0.21.0` to turn this off by default.  Again, my apologies on letting this slip through.

**Comment by ArondeParon** [comment]
No worries @mikeerickson! Can't catch every bug at once! 👍 

Thanks for the quick fix.
